src/directives/compose_area.ts

```diff
@@ -202,7 +202,7 @@ export default [
                         // Check for max length
                         const byteLength = (new TextEncoder().encode(text)).length;
                         if (byteLength > scope.maxTextLength) {
-                            const pieces: string[] = stringService.byteChunk(text, scope.maxTextLength, 50);
+                            const pieces: string[] = stringService.byteChunk(text, scope.maxTextLength * 0.85, 50);
```

Comment:
Eh?

Reply:
Did you read the commit message?

Reply:
(A `Math.floor` call is missing though...)

Reply:
Umm, ok, but since it's a workaround, perhaps add a comment?

Reply:
Ok, fair point.